examples/napari_viewer.py

```diff
@@ -1,10 +1,10 @@
 """Example showing how to access the current napari viewer
```

Comment:
I get a future warning in this example, possible because it hasn't moved over to the new syntax yet

```python-traceback
(base) czirwc1macos2701:magicgui nsofroniew$ python examples/napari_viewer.py
/Users/nsofroniew/GitHub/magicgui/magicgui/widgets/_bases.py:658: FutureWarning:

As of magicgui 0.2.0, a `choices` callable may accept only a single positional
argument (an instance of `magicgui.widgets.CategoricalWidget`), and must return
an iterable (the choices to show). Function 'get_viewers' accepts 2 arguments.
In the future, this will raise an exception.

  FutureWarning,
/Users/nsofroniew/GitHub/magicgui/magicgui/function_gui.py:189: FutureWarning:

Creating a widget instance with `my_function.Gui()` is deprecated,
the magicgui decorator now returns a widget instance directly, so you
should simply use the function itself as a magicgui widget, or call
`my_function.show(run=True/False)` to run the application.
In a future version, the `Gui` attribute will be removed.

  FutureWarning,
```

Reply:
going to remove this altogether since it's confusing to know whether this is necessary or not (this sort of napari type lookup will be in `napari.utils._magicgui`)